examples/monolith/main.tf

```diff
@@ -0,0 +1,302 @@
+  source                                   = "../.."
+  resource_group_id                        = module.resource_group.resource_group_id
+  region                                   = var.region
+  tags                                     = var.cluster_resource_tags
+  cluster_name                             = "${var.prefix}-${var.cluster_name}"
+  force_delete_storage                     = true
+  use_existing_cos                         = true
+  existing_cos_id                          = module.monolith_add_ons.cos_instance_id
+  vpc_id                                   = module.monolith_add_ons.vpc_id
+  vpc_subnets                              = local.vpc_subnets
+  ocp_version                              = var.openshift_version
+  worker_pools                             = local.worker_pools
+  access_tags                              = var.access_tags
+  ocp_entitlement                          = var.ocp_entitlement
+  additional_lb_security_group_ids         = var.additional_lb_security_group_ids
+  additional_vpe_security_group_ids        = var.additional_vpe_security_group_ids
+  addons                                   = local.addons
+  allow_default_worker_pool_replacement    = var.allow_default_worker_pool_replacement
+  attach_ibm_managed_security_group        = var.attach_ibm_managed_security_group
+  cluster_config_endpoint_type             = var.cluster_config_endpoint_type
+  cbr_rules                                = var.ocp_cbr_rules
+  cluster_ready_when                       = var.cluster_ready_when
+  custom_security_group_ids                = var.custom_security_group_ids
+  disable_outbound_traffic_protection      = var.allow_outbound_traffic
+  disable_public_endpoint                  = !var.allow_public_access_to_cluster_management
+  enable_ocp_console                       = var.enable_ocp_console
+  ignore_worker_pool_size_changes          = var.ignore_worker_pool_size_changes
+  kms_config                               = module.monolith_add_ons.kms_config
+  manage_all_addons                        = var.manage_all_addons
+  number_of_lbs                            = var.number_of_lbs
+  pod_subnet_cidr                          = var.pod_subnet_cidr
+  service_subnet_cidr                      = var.service_subnet_cidr
+  verify_worker_network_readiness          = var.verify_worker_network_readiness
+  worker_pools_taints                      = var.worker_pools_taints
+  enable_secrets_manager_integration       = var.enable_secrets_manager_integration
+  existing_secrets_manager_instance_crn    = module.monolith_add_ons.secrets_manager_crn
+  secrets_manager_secret_group_id          = var.secrets_manager_secret_group_id != null ? var.secrets_manager_secret_group_id : (var.enable_secrets_manager_integration ? module.secret_group[0].secret_group_id : null)
+  skip_ocp_secrets_manager_iam_auth_policy = var.skip_ocp_secrets_manager_iam_auth_policy
+}
+
+resource "terraform_data" "delete_secrets" {
```

Comment:
this should be part of the module is this is a core capability as opposed to being left in example